Core.DomainModel/ItSystemUsage/ItSystemUsage.cs

```diff
@@ -628,6 +628,84 @@ public Maybe<OperationError> UpdateOrganizationalUsage(IEnumerable<OrganizationU
             return Maybe<OperationError>.None;
         }
 
+        private Maybe<ItSystemUsageOrgUnitUsage> GetOrganizationUnitUsage(int organizationUnitId)
+        {
+            return UsedBy.FirstOrDefault(ub => ub.OrganizationUnit.Id == organizationUnitId).FromNullable();
+        }
+
+        public Maybe<OperationError> RemoveResponsibleOrganizationUnit()
+        {
+            return UpdateOrganizationalUsage(GetUsedByOrganizationUnits(), Maybe<OrganizationUnit>.None);
+        }
+
+        public Maybe<OperationError> RemoveUsedByUnit(int unitId)
+        {
+            var selectedUnit = GetOrganizationUnitUsage(unitId);
+            if (selectedUnit.IsNone)
+                return new OperationError($"Unit with id: {unitId} was not found", OperationFailure.NotFound);
+
+            var remainingUnits = GetUsedByOrganizationUnits().Where(x => x.Id != unitId);
+            Maybe<OrganizationUnit> responsibleUnit = ResponsibleUsage?.OrganizationUnit;
+
+            if (ResponsibleUsage?.OrganizationUnit?.Id == unitId)
```

Comment:
why not use the responsible unit you just extracted...
`if(responsibleUnit.Select(unit=>unit.Id == unitId).GetValueOrFallback(false))`